sniffer/main.c

```diff
@@ -72,12 +77,14 @@ void dump_pkt(gnrc_pktsnip_t *pkt)
  */
 void *rawdump(void *arg)
 {
-    (void)arg;
-    msg_t msg;
+    msg_t msg_q[RAWDUMP_MSG_Q_SIZE];
```

Comment:
considering its size this should be moved up (outside this function) and made `static` to save RAM during runtime, obviously at the expense of binary size

Reply:
Even with this PR the stack usage of this thread is well below half of the allocated space on `samr21-xpro` (since it's only using one function with clear paths this is rather straight forward to measure, see below), so let's save the extra RAM and rather focus (in later PR) of reducing the stack size using the [new stack sizes we got](https://github.com/RIOT-OS/RIOT/pull/9356).

(compiled with `DEVELHELP=1` to get stack canary for usage, and waiting for at least one packet to get sniffed)

```
2018-06-18 17:26:03,620 - INFO # ps
2018-06-18 17:26:03,628 - INFO # 	pid | name                 | state    Q | pri | stack  ( used) | base addr  | current     
2018-06-18 17:26:03,636 - INFO # 	  - | isr_stack            | -        - |   - |    512 (  148) | 0x20000000 | 0x200001b8
2018-06-18 17:26:03,644 - INFO # 	  1 | idle                 | pending  Q |  15 |    256 (  152) | 0x2000052c | 0x20000594 
2018-06-18 17:26:03,653 - INFO # 	  2 | main                 | running  Q |   7 |   1536 (  692) | 0x2000062c | 0x20000a54 
2018-06-18 17:26:03,661 - INFO # 	  3 | at86rf2xx            | bl rx    _ |   2 |   1024 (  380) | 0x200012cc | 0x200015cc 
2018-06-18 17:26:03,669 - INFO # 	  4 | rawdump              | bl rx    _ |   6 |   1536 (  668) | 0x20000ccc | 0x20001124 
2018-06-18 17:26:03,675 - INFO # 	    | SUM                  |            |     |   4864 ( 2040)
```

Reply:
okay fine